src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -247,27 +263,11 @@ internal class ConnectionManager : IConnectionManager {
                 }
             }
 
-            if (!string.IsNullOrEmpty(connectionInfo?.Path) && await TrySwitchBrokerAsync(connectionInfo)) {
-                return;
-            }
-
-            var connection = RecentConnections.FirstOrDefault(c => !c.IsRemote);
```

Comment:
So we don't try to get user connected to anything if we failed to connect him to the last active connection?